specification/cognitiveservices/data-plane/ImmersiveReader/stable/v1.0/ImmersiveReader.json

```diff
@@ -0,0 +1,83 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Immersive Reader API v1.0",
+		"version": "1.0"
+	},
+	"x-ms-parameterized-host": {
+		"hostTemplate": "{Endpoint}/immersivereader/v1.0",
+		"useSchemePrefix": false,
+		"parameters": [{
+				"$ref": "#/parameters/Endpoint"
+			}
+		]
+	},
+	"securityDefinitions": {
+		"apiKeyHeader": {
+			"type": "apiKey",
+			"name": "Ocp-Apim-Subscription-Key",
+			"in": "header"
+		}
+	},
+	"security": [{
+			"apiKeyHeader": []
+		}
+	],
+	"paths": {
+		"/getcontentmodelforreader": {
+			"post": {
+				"description": "Creates an Immersive Reader session",
+				"operationId": "post-getcontentmodelforreader",
+				"summary": "/getcontentmodelforreader - POST",
+				"parameters": [{
+						"name": "MS-Int-AppId",
+						"in": "header",
+						"required": true,
+						"type": "string"
+					}, {
+						"name": "X-CorrelationId",
+						"in": "header",
+						"required": true,
+						"type": "string"
+					}, {
+						"name": "X-MS-DeviceId",
+						"in": "header",
+						"required": true,
+						"type": "string"
+					}, {
+						"name": "X-UserSessionId",
+						"in": "header",
+						"required": true,
+						"type": "string"
+					}, {
+						"name": "body",
```

Comment:
Why do you have an empty body parameter?
If this is indeed a GET, then why is it modeled as a POST request?

Reply:
It is a Post. It does require json data in the body. Was not sure whether the data format needed to be documented or not. It didn't seem necessary to get the API working in APIM.